e2e/e2e_test.go

```diff
@@ -1,19 +1,21 @@
 // +build e2e
 
-package main
+package e2e
```

Comment:
All other projects I know of have an `integration/test` directory. We should align this here. Feel free to do this in a separate PR.